tests/integration/stepfunctions/templates/timeouts/statemachines/service_lambda_wait_with_timeout_seconds_path.json5

```diff
@@ -0,0 +1,16 @@
+{
+  "Comment": "TIMEOUT_ON_TASK_SERVICE_LAMBDA_INVOKE_WITH_PATH",
+  "StartAt": "Start",
+  "States": {
+    "Start": {
+      "Type": "Task",
+      "TimeoutSecondsPath": "$.TimeoutSecondsValue",
```

Comment:
Please correct me if I'm wrong but I think I didn't see a test that has both a `TimeoutSeconds` as well as `TimeoutSecondsPath` field set. Might be good to add :thinking: